README.md

```diff
@@ -1,153 +1,7 @@
-# Message Semantics
-Structure of a routing packet (`M` in the diagrams)
-```rust
-    struct Message {
-        id: String, //should be formatted as [source_chain]:[unique identifier], i.e. Ethereum:0x74ac0205b1f8f51023942856145182f0e6fdd41ccb2c8058bf2d89fc67564d56:0
-        source_address: String,
-        source_chain: String,
-        destination_address: String,
-        destination_chain: String,
-        payload_hash: HexBinary
-    }
-```
-# High Level Architecture
-Incoming Flow:
-```mermaid
-flowchart TD
-subgraph Axelar
-	G1{"Gateway"}
-    G2{"Gateway"}
-	Vr{"Aggregate Verifier"}
-	Vo{"Voting verifier"}
-	R{"Router"}
-    S{"Service Registry"}
-end
+# Axelar Amplifier
 
-Relayer --"VerifyMessages([M1,M2])"-->G1
-G1 --"VerifyMessages([M1,M2])"--> Vr
-Vr --"VerifyMessages([M1,M2])"--> Vo
-Vo --"GetActiveWorkers"--> S
-Workers --"Vote(poll_id, votes)"--> Vo
+The Interchain Amplifier enables developers to permissionlessly set up connections to the Axelar network. Developers gain access to Axelar's interconnected network of chains and can "amplify" their resources by paying the cost equivalent to developing only one connection. They can establish connections between new ecosystems or existing chains to add new network properties, such as improved security or better delivery and availability.
```

Comment:
I don't think it looks good for our project's high level README to be so sparse. Most active open source projects have a detailed README. It's the landing page for our project, and I think it's important to have something substantial here. We probably want sections on how to build, and I also preferred having at least the short descriptions of each contract here. We should probably have a quick blurb about ampd too. The sequence diagrams and flow charts can be moved into a separate folder, since they are more detailed. But I would like that anyone clicking on our github link can get an idea of what the project is and how it works at a very basic level from just the high level README.